### PR TITLE
Require to run test and format in README

### PR DESCRIPTION
**Pull request by ChuanqiXu9**
## Why

In previous document, we don't require developer to run test and format before committing, which is necessary.

### CODE REVIEW

**Comment by qicosmos** [comment]
What clang-format version not mentioned.

**Comment by ChuanqiXu9** [comment]
> What clang-format version not mentioned.

The actual behavior is controlled by `.clang-format`. Although it is possible that higher version would have other option which didn't show up in current `.clang-format` and the option would break current format style. The possibility is not very high (clang-format is maintained well). Based on this, I prefer to not constraint user more.

**Review by qicosmos — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -69,9 +69,10 @@ After installing and reading [Lazy](./docs/docs.en/Lazy.md) to get familiar with
 
 # How to Contribute
 1. Read the [How to fix issue](./docs/docs.en/HowToFixIssue.md) document firstly.
-2. Create a PR, fill in the PR template.
-3. Choose one or more reviewers from contributors: (ChuanqiXu9, RainMark, foreverhy, qicosmos).
-4. Approved and merged.
+2. Run tests and `diff-clang-format HEAD^` locally for the change.
```
[the remark this answers is not in the document]
Comment (ChuanqiXu9) [reply README.md:72]: My bad. I would fix it.

**Pull request merged**